packages/schema-utils/src/index.ts

```diff
@@ -15,6 +15,12 @@ class CustomTypeBuilder extends Mixin(
 export function Validate<T extends TSchema>(schema: T, value: unknown): value is Static<T> {
     return Value.Check(schema, value);
 }
+export function Assert<T extends TSchema>(schema: T, value: unknown): asserts value is Static<T> {
+    const result = Value.Check(schema, value);
+    if (!result) {
+        throw new Error('Invalid value');
```

Comment:
👍 maybe, isn't there a way how to be more specific in this error message?
also as a next step - in connect package we are trying to have "typed errors", so maybe this function could have a wrapper inside connect that would throw a typed error defined in connect.

Reply:
Yes, the goal shoud probably be to throw the same exceptions as in the current validation logic: 
https://github.com/trezor/trezor-suite/blob/b38f6b1414048ad446309b0375e5b1079441f009/packages/connect/src/api/common/paramsValidator.ts#L21-L93